docker-compose.yml

```
@@ -28,6 +28,7 @@ services:
     volumes:
       - ./src/backend:/usr/local/src/backend
       - $HOME/.aws:/root/.aws:ro
+      - ./taui/assets:/usr/local/src/backend/static/assets
```

```suggestion
      - ./taui/assets:/usr/local/src/backend/static/assets
      - ./taui/index.html:/usr/local/src/backend/static/index.html
```

Unfortunately, copying `index.html` as a command didn't work for me; however, mounting it as its own volume did. I'm not sure whether this is best practice but it makes sense to me that this file could persist across Docker containers like the rest of the stuff in `taui/assets`.

---

I'm thinking I must have already had the `index.html` file in my `/static` folder after trying to mount from before, so it was able to work for me locally but not actually in practice. This way makes sense to me and works well, so just pushed that change!